images/release/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN yum install -y hg golang golang-pkg-darwin golang-pkg-windows && yum clean a
 ENV GOPATH /go
 
 # Get the code coverage tool and godep
-RUN go get code.google.com/p/go.tools/cmd/cover github.com/tools/godep github.com/golang/lint/golint
+RUN if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
```

Comment:
The old location is still valid and recommended for Go 1.2/1.3. Starting in 1.4 the new location must be used. Related upstream issue: https://github.com/GoogleCloudPlatform/kubernetes/issues/2728. 

So I tried to use the same strategy used in `travis.yml`, but I think we can safely remove the condition and just use the new location if we are guaranteed to always get 1.4 from `yum install golang`.